l10_ir_accounting/views/account_group_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="accoutn_group_action" model="ir.actions.act_window">
+            <field name="name">گروه حساب</field>
+            <field name="res_model">account.group</field>
+            <field name="view_mode">tree,form</field>
+            <field name="search_view_id" ref="account.view_account_group_search"/>
+            <field name="view_id" ref="account.view_account_group_tree"/>
+            <field name="help" type="html">
+                <p class="o_view_nocontent_smiling_face">
```

Comment:
if you are re-defining this action/menu to translate strings better using i18n

or why it was redefined?

Reply:
https://github.com/OCA/account-financial-tools/tree/14.0/account_menu
These changes are no longer needed
Removed from code